privacyidea/api/before_after.py

```diff
@@ -226,7 +233,7 @@ def after_request(response):
     """
     # In certain error cases the before_request was not handled
     # completely so that we do not have an audit_object
-    if "audit_object" in g:
+    if "audit_object" in g and bool(g.audit_object.audit_data):
```

Comment:
We could shorten this to ``if "audit_object" in g and g.audit_object.audit_data:`` as the ``bool()`` isn't necessary here. What do you think?

Reply:
Done in 541cfc7c

Reply:
lgtm. Thanks!